src/taskgraph/run-task/run-task

```diff
@@ -1053,9 +1054,14 @@ def maybe_run_resource_monitoring():
     monitor_process.start()
     return process
 
+def _display_pyhton_version():
```

Comment:
I don't think it's worth pulling this out into a new function, let's just keep it in-line in `main`.

Reply:
Thank you for your review. I did it on the advice of @JohanLorenzo . The objective was to isolated it for the unit tests. I can indeed send it back to the main function

Reply:
Ah, my mistake I didn't notice it was being used by the test. That is fine then!

I do notice you have a typo in the function name though, `s/pyhton/python`. Please fix :)